docs/src/app/components/pages/home.jsx

```diff
@@ -8,6 +8,7 @@ const { StylePropable, StyleResizable } = Mixins;
 const { Colors, Spacing, Typography } = Styles;
 const ThemeManager = Styles.ThemeManager;
 const DefaultRawTheme = Styles.LightRawTheme;
+require('github-markdown-css/github-markdown.css');
```

Comment:
This statement throws an error on my side. Not sure if you can import CSS like that. Might have to include in the `index.html`